src/catchup/ApplyBucketsWork.cpp

```diff
@@ -163,8 +163,12 @@ ApplyBucketsWork::advance(BucketApplicator& applicator)
         return;
     }
 
+    auto& meter =
```

Comment:
minor: it's probably a bit more consistent and efficient to just have this meter as a member variable

Reply:
fixed (this is carried on over in the bulk-operations PR; I'll close this one).